signer/src/bitcoin/utxo.rs

```diff
@@ -2940,50 +2970,100 @@ mod tests {
             accept_threshold: 11,
             num_signers: 15,
             sbtc_limits: SbtcLimits::default(),
+            max_deposits_per_bitcoin_tx: DEFAULT_MAX_DEPOSITS_PER_BITCOIN_TX,
         };
 
         let transactions = requests.construct_transactions().unwrap();
-        assert_eq!(transactions.len(), 25);
+        assert_eq!(transactions.len(), 15);
```

Comment:
IIUC this test was to check that we don't create a chain of more than `MAX_TX_PER_BITCOIN_BLOCK` txs, if so we should probably change the votes in deposits and withdrawals to force creating singleton transactions (eg, adding a `| 0b111` and incrementing both shifts by 3).

Reply:
Hmmm, I thought that it was to check that the vsize was capped by `MEMPOOL_MAX_PACKAGE_SIZE`, but maybe it's both? Yeah, let's check both.

Reply:
Oh whoops, it's supposed to check that `MAX_TX_PER_BITCOIN_BLOCK` is capped.

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/1181/commits/22eb34651dff1e968dc9118bd2306672d1e737a4.

Reply:
nit: the comment `we should generate 60 distinct transactions` is no longer true, given `SbtcRequests` allow for up to 4 votes against it should generate only ~30 transactions.

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/1181/commits/4641ddcd44ff00497e826dfa8bbb51115aee2459